src/editor/editor_comment.hpp

```diff
@@ -0,0 +1,50 @@
+#ifndef EDITOR_COMMENT_HPP
+#define EDITOR_COMMENT_HPP
```

Comment:
Missing copyright header, and also, follow the include guard convention used throughout the source.